lib/website-infra-aaron-stack.ts

```diff
@@ -59,8 +72,9 @@ export class WebsiteInfraAaronStack extends cdk.Stack {
             {
               isDefaultBehavior: true,
               compress: true,
+              defaultTtl: cdk.Duration.seconds(900),
+              minTtl: cdk.Duration.seconds(60),
```

Comment:
Adding default cache headers to all of the content that is served from CloudFront will mean that the content will be cached. Personally I would turn this on.

It is also possible to add configuration that overides the headers that are added to the response.

Reply:
^ Such headers that are normally added are the "security ones"